runtime.txt

```diff
@@ -1 +1 @@
-python-3.6.3
+python-3.10.17
```

Comment:
Anything preventing us from upgrading to a later version (3.13)?

Reply:
might be heroku specific, not sure though. but let this in and upgrade in another PR later?